apis/tempo/v1alpha1/microservices_types.go

```diff
@@ -128,6 +134,21 @@ type Resources struct {
 	Total *corev1.ResourceRequirements `json:"total,omitempty"`
 }
 
+type SearchSpec struct {
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Max search time range allowed"
+	MaxSearchDuration string `json:"maxSearchDuration,omitempty"`
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Limit used for search requests if none is set by the caller"
+	DefaultResultLimit int `json:"defaultResultLimit,omitempty"`
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="The maximum allowed value of the limit parameter on search requests"
+	MaxResultLimit int `json:"maxResultLimit,omitempty"`
```

Comment:
what is the default value?

Reply:
The default value is the value indicated in the tempo config docs , in this case this limit is disabled ( value = 0)

Reply:
Is this related to `DefaultResultLimit` ? If so it makes sense to colocate them and crosslink them in the godoc.

Reply:
Not really, this is the max number of traces you can fetch from tempo, I think the propose of this is  to artificially limit the results in order to prevent abuses, (queries that returns a lot of traces which could overload the system)

Reply:
I have the same comment as for `DefaultResultLimit` we should mention what it actually limits - traces/spans.